Website/src/rendering/src/components/Checkout/CheckoutDetails.tsx

```diff
@@ -23,8 +17,10 @@ const CheckoutDetails = (): JSX.Element => {
         <PanelBillingAddress />
         <PanelPayment />
         <div>
-          <PanelComments orderComments={comments} onEditComments={handleEditComments} />
-          <CheckoutSummary orderComments={comments} />
+          <PanelComments />
+          <Link href="/shop/checkout/order-review">
```

Comment:
We still want the `<CheckoutSummary>` component in the checkout page. It updates as we select different shipping methods. This is also the component that has the logic to allow continuing to the next step (order review) only if all the checkout sections have been filled.

We want the button at the bottom of the `CheckoutSummary` component to be replaced by the "Review Order" button while we are on the checkout page. We want a "Place your order" button while on the "Review order" page.